packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -49,7 +55,14 @@
             <component
                 :is="errorBoundary"
                 tier="3">
-                <slot name="new-label" />
+                <!-- TODO - we want to translate this within the component using i18n.
```

Comment:
To confirm - is this work listed as `TODO` captured in an internal ticket?

Reply:
Yeah completely, the plan is to add i18n as a stretch goal after initial delivery due to f-globalisation work needed first (which I think Core web are owning)